test/bench/etc/run_granular_benchmarks.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+/**
+ * This script runs all benchmarks in test/bench/lib/granular and merges their output
+ */
+
+const cp = require('child_process');
+const fs = require('fs/promises');
+const path = require('path');
+const { Task } = require('dbx-js-tools/packages/bson-bench');
+
+const BENCHMARK_REGEX = /(.*)\.bench\.js$/;
+const BENCHMARK_PATH = path.resolve(`${__dirname}/../lib/granular`);
+const DOCUMENT_ROOT = path.resolve(`${__dirname}/../documents`);
+const LIBRARY_PATH = path.resolve(`${__dirname}/../../../.`);
+(async () => {
+  // HACK : run one dummy task with the local bson to ensure it's available for subsequent suites
```

Comment:
If this is a hack what is the non-hack way to accomplish this?

Reply:
The best way to go about fixing this would be to expose the `Package` type from the bson-bench library repository and run `Package.install`.

Reply:
Can we allude to that in the comment or file a ticket to expose that?

Reply:
Will file a ticket for this and reference in the comment